source/mapping-rules/mapping-rule-options/mapping-rule-filters.txt

```diff
@@ -0,0 +1,135 @@
+.. _rm-mapping-filters:
+
+====================
+Mapping Rule Filters
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In a migration project, you can use mapping rule javascript expressions to filter mapping rules based on document values. Mapping rule filters are available to all mapping rule types.
+
+Mapping rule options allow you to:
+
+- Filter mapping rules based on logical conditions.
+- Limit the number of documents written to a table.
+- Access complex and nested document structure for filtering.
+
+
+Example
+-------
+
+You can apply one filter per mapping rule. Each filter can have a javascript expression that when evaluated to ``true`` filters documents for migration.
+
+Below are examples of input documents and mapping rule javascript expressions that filter the data.
+
+
+Filter Based on Field Value
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example filters a document based on a single field value.
+
+Relational input:
+
+.. list-table::
+    :header-rows: 1
+
+    * - customer_id
+      - company_name
+      - contact_name
+      - contact_title
+      - address
+      - city
+      - region
+      - postal_code
+      - country
+      - phone
+      - fax
+
+
+    * - ALFKI
+      - Alfreds Futterkiste
+      - Maria Anders
+      - Sales Representative
+      - Obere Str. 57
+      - Berlin
+      - ``null``
+      - 12209
+      - Germany
+      - 030-0074321
+      - 030-0076545
+
+    * - ANATR
+      - Ana Trujillo Emparedados y helados
+      - Ana Trujillo
+      - Owner
+      - Avda. de la Constitución 2222
+      - México D.F.
+      - ``null``
+      - 05021
+      - Mexico
+      - (5) 555-4729
+      - (5) 555-3745
+
+    * - ANTON
+      - Antonio Moreno Taquería
+      - Antonio Moreno
+      - Owner
+      - Mataderos  2312
+      - México D.F.
+      - ``null``
+      - 05023
+      - Mexico
+      - (5) 555-3932
+      - ``null``
+
+
+Filter Expression:
+
+.. code-block:: none
+
+   columns["country"] == "Mexico"
+
+
+MongoDB output:
+
+.. code-block:: none
+   :copyable: false
+
+   [
+       {
+           "_id": {
+           "$oid": "661fed11aa6fa8b49e28ac03"
+           },
```

Comment:
The example is better now, but if this was an array it wouldn't have _id fields. Can we remove these for clarity?